rules/linux/lnx_process_discovery.yml

```diff
@@ -0,0 +1,22 @@
+title: Process Discovery
+id: 4e2f5868-08d4-413d-899f-dc2f1508627b
+status: stable
+description: Detects process discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://attack.mitre.org/techniques/T1057/
+logsource:
+    product: linux
+detection:
+    keywords:
+        - commands|contains:
+          - 'ps '
+          - 'top'
```

Comment:
could you please use [this](https://github.com/Neo23x0/sigma/pull/1044#discussion_r502049487) recommendation here as well?

Reply:
Hello, @omergunal I believe it's better to use ProcessName here, because it that would be a part of commandline, it would produce lots of FP. Could you please change that?

Reply:
thank you!